scripts/extract_implications.lean

```diff
@@ -1,4 +1,20 @@
-import equational_theories.ParseImplications
+import Lean.Meta.Basic
+import Lean.Util
 
-unsafe def main (_args : List String) : IO Unit := do
-  println! (Lean.toJson (← generateOutput)).pretty
+import equational_theories.Result
+
+open Lean
+
+unsafe def main (args : List String) : IO Unit := do
+  let module := ← match args with
+    | [] => pure `equational_theories
+    | [mod] => pure mod.toName
+    | _ => throw <| IO.userError "usage: extract_impliations MODULE"
```

Comment:
Perhaps you can use the CLI from #102

Reply:
could make sense in a later PR. For now, I want to focus on getting the core functionality here landed.